whois-api/pom.xml

```diff
@@ -44,6 +44,16 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>net.ripe.db</groupId>
+            <artifactId>whois-nrtm4-client</artifactId>
```

Comment:
why these are needed?

Reply:
If I don't put that dependency there the module doesn't run and the schedule task doesn't start